k8s/namespaces/cpo/case-payment-orders-api/ithc.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: case-payment-orders-api
+spec:
+  values:
+    java:
+      environment:
+        S2S_AUTHORIZATIONS_XUI_ID: xui_webapp
```

Comment:
you seem to be duplicating this in multiple places but it's already setup as this in values.yaml?

https://github.com/hmcts/hmcts-charts/blob/e30c9341fc03dc339382fedea7b231dcdee16474/stable/cpo-case-payment-orders-api/values.yaml#L30

duplicates (and more places)
https://github.com/hmcts/hmcts-charts/blob/e30c9341fc03dc339382fedea7b231dcdee16474/stable/cpo-case-payment-orders-api/values.aat.template.yaml#L7

https://github.com/hmcts/hmcts-charts/blob/e30c9341fc03dc339382fedea7b231dcdee16474/stable/cpo-case-payment-orders-api/values.preview.template.yaml#L17

```suggestion
```